website/docs/modules/sources.html.markdown

```diff
@@ -47,22 +47,50 @@ Updates for file paths are automatic: when "downloading" the module using the [g
 The [Terraform Registry](https://registry.terraform.io) is an index of modules
 written by the Terraform community.
 The Terraform Registry is the easiest
-way to get started with Terraform and to find modules to start with.
-The registry is integrated directly into Terraform:
+way to get started with Terraform and to find modules.
+
+The registry is integrated directly into Terraform. You can reference any
+registry module with a source string of `<NAMESPACE>/<NAME>/<PROVIDER>`. Each
+module's information page on the registry includes its source string.
 
 ```hcl
 module "consul" {
   source = "hashicorp/consul/aws"
+  version = "0.1.0"
```

Comment:
Our usual HCL style calls for lining up the equals signs here. We try to keep the examples in the docs consistent with what `terraform fmt` would produce so users don't see changes if they run that command.